/jds/bin/diff -uprN lynx2-8-6.old/lynx.man lynx2-8-6/lynx.man
--- lynx2-8-7.old/lynx.man	2009-02-20 18:50:31.235553000 +0000
+++ lynx2-8-7/lynx.man	2009-02-20 18:54:07.551070000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" $LynxId: lynx.man,v 1.86 2009/07/05 20:59:11 tom Exp $
 .nr N -1
 .nr D 5
@@ -1125,6 +1126,21 @@ after subscribing.
 Unsubscribe by sending email to <lynx-dev-request@nongnu.org> with
 "unsubscribe lynx-dev" as the only line in the body of your message.
 Do not send the unsubscribe message to the lynx-dev list, itself.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWlynx
+Interface stability^Volatile
+.TE
+.sp
 .SH SEE ALSO
 .hy 0
 \fIcatgets\fR(3),
@@ -1187,3 +1203,5 @@ Shrewsbury, Massachusetts 01545
 Thomas E. Dickey
 .br
 <dickey@invisible-island.net>
+...\" LSARC 2008/700 lynx 2.8.6
+...\" PSARC 2000/488 Solaris/Linux Commands Compatibility
